testsuite/features/secondary/min_salt_software_states.feature

```diff
@@ -40,13 +40,14 @@ Feature: Salt package states
     Then the system should have a base channel set
 
   Scenario: Remove a package through the UI
-    Then I follow "States" in the content area
+    When I follow "States" in the content area
     And I follow "Packages"
     And I follow "Search"
     And I should see a "Package States" text
     And I list packages with "dummy"
-    Then I should see a "milkyway-dummy" text
-    And "milkyway-dummy" should be installed on "sle_minion"
+    And I wait for "60" seconds
```

Comment:
Uf, avoid explicit waits as much as possible.
Use a "wait until I see" and play with the timeout. But don't strictly wait always 60 seconds, because that depends very mucho on the environment we run it.

Reply:
This was the Net timeout issue, there was nothing I could do outside of that since waiting for any element resulted in the same failure
However it might be only a PR pipeline issue and it could not be necessary in other CIs

Reply:
If it's a problem on the PR pipeline, we need to find a way to fix it at that level, but waiting 60 seconds idle for sure is not an option.

Reply:
I'll reproduce it to see how we can fix it, but there was absolutely no logs to identify the issue, and the ajax transition did not work on it, it was really odd

Reply:
The timeout occurs when waiting for the text to appear and is completely different from the timeout expected when waiting the text: 
```
When(/^I wait until I see "([^"]*)" text$/) do |text|
  raise "Text '#{text}' not found" unless has_text?(text, wait: DEFAULT_TIMEOUT)
end
```
![image](https://user-images.githubusercontent.com/100688791/195813253-b7ce08d5-4a42-4273-9b24-ff7d97842d02.png)

Searching for more information online provided this page: https://www.exceptionalcreatures.com/bestiary/Net/ReadTimeout

It seems to be an issue that cannot be solved by waiting for an element of the page.
Keeping this failing results in the scenario failing after 400 seconds, when keeping this explicit wait makes it succeed after 120 seconds.

It is beyond my understanding of what is happening and would require time and help to identify what the whole problem is.
I will keep these explicit waits until the PR is ready to merge to avoid these scenarios failing at every test run, hopefully this issue will not appear on the main pipelines.